plugins/dell/fu-plugin-dell.c

```diff
@@ -822,11 +822,12 @@ fu_plugin_device_registered (FuPlugin *plugin, FuDevice *device)
 		if (fu_device_get_metadata_boolean (device, FU_DEVICE_METADATA_TBT_IS_SAFE_MODE)) {
 			g_autofree gchar *vendor_id = NULL;
 			g_autofree gchar *device_id = NULL;
-			vendor_id = g_strdup_printf ("TBT:0x%04d", 0x00d4);
-			device_id = g_strdup_printf ("TBT-%04d%04d", 0x00d4,
+			vendor_id = g_strdup_printf ("TBT:0x%04x", 0x00d4);
+			device_id = g_strdup_printf ("TBT-%04x%04x", 0x00d4,
```

Comment:
I think this needs to be `X` to get upper case hex.

Reply:
yep, good catch.  i'm a bit baffled why the GUID was correct without this, but the DeviceVendorID is definitely the wrong case.

Reply:
Looks like it's the device GUID is lowercase, but `DeviceVendorID `is upper case.